simbev/main_simbev.py

```diff
@@ -79,6 +79,9 @@ def run_simbev(region_ctr, region_id, region_data, cfg_dict, charge_prob,
     ]
     count_cars = 0
 
+    region_path = main_path.joinpath(str(region_id))
+    region_path.mkdir(exist_ok=True)
```

Comment:
I prefare `os.makedirs()` over `path.mkdir()`. See [this](https://stackoverflow.com/a/13819575/13491957) answer on SO.